toolkit/tools/internal/exe/exe_test.go

```diff
@@ -0,0 +1,58 @@
+package exe
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseListArgument(t *testing.T) {
+	tests := []struct {
+		name     string
+		input    string
+		expected []string
+	}{
+		{
+			name:     "empty input",
+			input:    "",
+			expected: []string{},
+		},
+		{
+			name:     "single value",
+			input:    "value",
+			expected: []string{"value"},
+		},
+		{
+			name:     "multiple values",
+			input:    "value1 value2 value3",
+			expected: []string{"value1", "value2", "value3"},
+		},
+		{
+			name:     "leading/trailing spaces",
+			input:    "  value1  value2  value3  ",
+			expected: []string{"value1", "value2", "value3"},
+		},
+		{
+			name:     "extra spaces",
+			input:    "value1  value2   value3",
+			expected: []string{"value1", "value2", "value3"},
+		},
+		{
+			name:     "tabs",
+			input:    "value1\tvalue2\tvalue3",
+			expected: []string{"value1", "value2", "value3"},
+		},
+		{
+			name:     "mixed whitespace",
+			input:    "value1\tvalue2  value3",
+			expected: []string{"value1", "value2", "value3"},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
```

Comment:
Not familiar with golang testing, so just making sure. This would show exactly which testcase failed and which passed?

Reply:
Hah, actually had this chat with Tobias as well. This seems to be the "recommended" form for tests.  Each sub-test will have its name printed if it fails, although unfortunately the line numbers refer back to the call to `Run()` so a bit of extra leg work is needed.